systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -202,12 +206,13 @@ public static boolean checkLogForJSONFormat(Map<String, String> pods, String con
         boolean isJSON = false;
 
         for (String podName : pods.keySet()) {
-            String logs = kubeClient().logs(podName, containerName).replaceFirst("([^{]+)", "");
-            try {
-                new JsonObject(logs);
+            String log = cmdKubeClient().exec(true, false, "logs", podName, "-c", containerName, "--tail=100").out();
+            log = log.replaceAll("[\n|\r]", "");
```

Comment:
Do you really mean that character class containing `\n`, `\r` and `|`, or did you just mean to include only `\n` and `\r`? If the latter that would be `[\n\r]`. If the former it might be better to use `[\n\r|]` to make it clearer that you really meant to include `|`.

Reply:
I meant to include `\n` and `\r`, I saw that's working regex for this. Because in the log we can find both of these and I wanted to change it to `""`

Reply:
Regex has different syntax for if you're trying a match one of a set of _characters_ (the "character class", where, among other possibilities, you spell out the chars within `[` and `]` with no separator, e.g. [`abc`]) or one of a set of other _regexes_ (the "group", where you spell out the subexpressions within `(` and `)` each separated with `|`, e.g. `(foo|bar)`). 

The regex you used will also match a `|`, which isn't what you meant. It probably doesm't make much difference in this particular case, but it's worth getting regexes right as in other cases it might make a big difference.

Reply:
Okay, thanks for the explanation, I'm gonna fix it :).